ui/app/components/clients/attribution.hbs

```diff
@@ -3,53 +3,32 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-{{! only show side-by-side horizontal bar charts if data is from a single, historical month }}
-<div
-  class={{concat "chart-wrapper" (if @isHistoricalMonth " dual-chart-grid" " single-chart-grid")}}
-  data-test-clients-attribution
->
+<div class="chart-wrapper single-chart-grid" data-test-clients-attribution={{this.noun}}>
   <div class="chart-header has-bottom-margin-m">
-    <h2 class="chart-title">Attribution</h2>
+    <h2 class="chart-title" data-test-attribution-title>{{capitalize this.noun}} attribution</h2>
     <p class="chart-description" data-test-attribution-description>{{this.chartText.description}}</p>
   </div>
-  {{#if this.barChartTotalClients}}
-    {{#if @isHistoricalMonth}}
-      <div class="chart-container-left" data-test-chart-container="new-clients">
-        <h2 class="chart-title">New clients</h2>
-        <p class="chart-description">{{this.chartText.newCopy}}</p>
-        <Clients::HorizontalBarChart
-          @dataset={{this.barChartNewClients}}
-          @chartLegend={{this.attributionLegend}}
-          @noDataMessage="There are no new clients for this namespace during this time period."
-        />
-      </div>
+  {{#if @attribution}}
+    <div
+      class={{concat (unless @attribution "chart-empty-state ") "chart-container-wide"}}
```

Comment:
I think this can be removed since we're inside the `{{#if @attribution}}` conditional
```suggestion
      class="chart-container-wide"
```

Reply:
oh hm.. maybe I borked this, I'll need to double check the logic. Because we still can get to an empty state (or we should)